packages/date-picker/test/basic.test.js

```diff
@@ -1,33 +1,9 @@
 import { expect } from '@esm-bundle/chai';
-import {
-  aTimeout,
-  click,
-  fixtureSync,
-  keyboardEventFor,
-  makeSoloTouchEvent,
-  oneEvent,
-  tap,
-} from '@vaadin/testing-helpers';
+import { aTimeout, click, fixtureSync, keyboardEventFor, oneEvent, tap } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import sinon from 'sinon';
 import '../src/vaadin-date-picker.js';
-import * as settings from '@polymer/polymer/lib/utils/settings.js';
-import { close, getFocusedCell, getOverlayContent, idleCallback, open } from './common.js';
-
-settings.setCancelSyntheticClickEvents(false);
```

Comment:
This is no longer needed here, because we enable it in `ElementMixin` since #3510